include/oneapi/tbb/version.h

```diff
@@ -17,15 +17,21 @@
 #ifndef __TBB_version_H
 #define __TBB_version_H
 
-#include "detail/_config.h"
-#include "detail/_namespace_injection.h"
+// Exclude all includes during .rc files compilation
+#ifndef RC_INVOKED
+    #include "detail/_config.h"
+    #include "detail/_namespace_injection.h"
+#else
+    #define __TBB_STRING_AUX(x) #x
+    #define __TBB_STRING(x) __TBB_STRING_AUX(x)
+#endif
 
 // Product version
 #define TBB_VERSION_MAJOR 2021
 // Update version
 #define TBB_VERSION_MINOR 4
 // "Patch" version for custom releases
-#define TBB_VERSION_PATCH 0
+#define TBB_VERSION_PATCH 1
```

Comment:
@omalyshe don't we need to update `TBB_INTERFACE_VERSION` to `12041` as well?

Reply:
According to the internal discussion we have to update the TBB_INTERFACE_VERSION as well. @dnmokhov Can you please update the PR?

Reply:
Updated.